kubernetes/example-config/pod.yml

```diff
@@ -6,7 +6,7 @@ metadata:
     app: github-nginx-cache
 spec:
   containers:
-    - image: azuredevx/github-nginx-cache:test
+    - image: azuredevx/github-nginx-cache:latest
```

Comment:
maybe add a warning to use latest only for dev purposes. for prod use a specific release tag